src/server/pfs/pfs.go

```diff
@@ -79,6 +84,10 @@ func (e ErrRepoExists) Error() string {
 	return fmt.Sprintf("repo %v already exists", e.Repo.Name)
 }
 
```

Comment:
Do we need to specify that the command needs to be re-run in this error? Or do we need to create another err to inform the user about inconsistent state? I don't think it's enough to specify this alone in the help text.

Reply:
No, this is primarily intended to block off operations on the deleted repo that are running in parallel with an ongoing deletion, not determine whether a delete operation needs to be rerun. We actually wouldn't be able to make that judgement here because the deletion could be running in parallel, and we don't actually know what operation would need to be resumed. All we know is that a repo is marked as deleted, therefore we should not be performing operations on it.